style.css

```diff
@@ -8,7 +8,92 @@ body {
 }
 
 .menu_items {
+  position: absolute;
+  left: 0;
+  background-color: #191765;
+  height: 100vh;
+  width: 100%;
+  z-index: 1;
+  top: 0;
+  list-style-type: none;
   display: none;
+  margin-left: 0;
+  padding-left: 0;
+}
```

Comment:
- [ ] Also consider adding the background image like figma design and consider adding blur effect also to it to match figma design.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/188514333-117e0f01-24b2-4d5e-9db2-e24cb4f11f90.png) | ![image](https://user-images.githubusercontent.com/53402600/188514300-f4e7d97e-2df0-44b3-914b-6474c146bc2a.png) |